packages/engine/Source/Scene/Model/Model.js

```diff
@@ -1339,6 +1344,39 @@ Object.defineProperties(Model.prototype, {
     },
   },
 
+  /**
+   * Enable Models to participate in scene verticalExaggeration.
+   *
+   * @memberof Model.prototype
+   *
+   * @type {boolean}
+   */
+  allowVerticalExaggeration: {
+    get: function () {
+      return this._allowVerticalExaggeration;
+    },
+    set: function (value) {
+      if (value !== this._allowVerticalExaggeration) {
+        this.resetDrawCommands();
+        //this._verticalExaggerationDirty = true;
```

Comment:
I assume we can drop this commented line?

Reply:
removed